hack/start

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o errtrace
+set -o pipefail
+set -o nounset
+
+doc() {
+    cat <<-EOF
+Starts a local docusaurus development server
+
+USAGE:
+   start
+
+EXAMPLES:
+    update-snippets
```

Comment:
d: I am not sure about this example? What does that tell us? I can't see that this would be used anywhere? Same applies to `start`. Does that have any consequence? Perhaps it's just my lack of bash knowledge. But i can't see that it is actually used. just looks to me like the main function is only called if any parameter is given? Otherwise this docs is outputed?

Reply:
sorry `update-snippets` is a copy paste error. yes it's a bit weird to have examples when there are no parameters but its just a conventional script scaffold to provide a help text like this